rhtap/summary.sh

```diff
@@ -34,6 +34,21 @@ function cosignTree() {
     cosign tree $URL
 }
 
+echo "SUMMARY_EYECATCHER_BEGIN"
+
+jq -n --arg base "$BASE_RESULTS" '
+  [ inputs ]' < <(
```

Comment:
(this also doesn't seem like it should work, since it uses the `--null-input` flag and collects inputs)